images/win/scripts/Tests/Vsix.Tests.ps1

```diff
@@ -0,0 +1,12 @@
+Describe "Vsix" {
+    $toolset = Get-ToolsetContent
+    $requiredVsixs = $toolset.visualStudio.vsix
+
+    $allPackages = (Get-VSSetupInstance | Select-VsSetupInstance -Product *).Packages

```

Comment:
Do we need call Select-VsSetupInstance -Product *?

Reply:
Please specify other approach

Reply:
according to https://raw.githubusercontent.com/Microsoft/vssetup.powershell/master/docs/VSSetup/Get-VSSetupInstance.md
Get-VSSetupInstance enumerates the VS Instances. I've got the next result of this cmdlet:
![image](https://user-images.githubusercontent.com/57713077/92947970-c0b43380-f469-11ea-996f-5a79578e4dac.png)
According to: https://raw.githubusercontent.com/Microsoft/vssetup.powershell/master/docs/VSSetup/Select-VSSetupInstance.md, we need to use Select-VSSetupInstance cmdlet for selecting related products basses on criteria